### PR TITLE
Progress chart

### PR DESCRIPTION
**Pull request by erundle**
Directive to create a basic progress chart

### CODE REVIEW

File: src/charts/progress/progress-chart.directive.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ * <ul style='list-style-type: none'>
+ * <li>.used   - number representing the amount used
+ * <li>.total  - number representing the total amount
+ * </ul>
+ *
+ * @param {object} config the chart configuration options<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.title  - the chart title
+ * <li>.units  - units to be displayed on the chart
+ * </ul>
```
Comment (jeff-phillips-18) [inline src/charts/progress/progress-chart.directive.js]: I think a config object is overkill here. Can we just use title and units parameters?

Comment (erundle) [inline src/charts/progress/progress-chart.directive.js]: Sure, will make that change.

File: test/charts/progress/progress.spec.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    expect(progressBar).toBe("50%");
+
+    title = angular.element(element).find('.progress-description').html();
+    expect(title).toBe("CPU Usage");
+
+    subTitle = angular.element(element).find('.progress-bar span').html();
+    expect(subTitle).toBe("8 of 16 MB");
+  });
+
+});
```
Comment (jeff-phillips-18) [inline test/charts/progress/progress.spec.js]: Could we add a test to verify updating the data, title or units updates the displayed components? Silly and easy, but it adds check.

File: test/charts/progress/progress.spec.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+  beforeEach(module(
+    'patternfly.charts',
+    'charts/progress/progress-chart.html'
+  ));
+
+  beforeEach(inject(function(_$compile_, _$rootScope_, _$timeout_) {
+    $compile = _$compile_;
+    $scope = _$rootScope_;
+    $timeout = _$timeout_;
```
Comment (jeff-phillips-18) [inline test/charts/progress/progress.spec.js]: I don't see $timeout being used

**Comment by serenamarie125** [comment]
Please hold off on this PR.  We need design and high fidelity first, and then evaluate if implementation matches.

**Pull request closed without merging**